tests/commands/base/test_download_url.py

```diff
@@ -39,6 +40,35 @@ def test_new_download_oneshot(base_command):
         assert f.read() == 'all content'
 
 
+def test_new_download_content_disposition(base_command):
+    base_command.requests = mock.MagicMock()
+    response = mock.MagicMock()
+    response.url = 'https://example.com/path/to/irrelevant.zip'
+    response.content = b'content'
+    response.iter_content.return_value = iter([response.content])
+    # We use strange case & whitespace in this sample header to ensure we're lenient in what we accept.
+    # https://tools.ietf.org/html/rfc6266 permits spaces, quotes, and strange case in these places.
+    response.headers = HTTPHeaderDict({
+        'content-disposition': 'Content-Disposition: ATTACHment; filename=    "something.zip"',
```

Comment:
This test seems to be trying to validate a bunch of things in one. Could we parameterize the test for a bunch of possible cases (including the RFC6266 case you flag in the implementation)?

Reply:
+1